src/OpenTelemetry.Instrumentation.EntityFrameworkCore/CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## Unreleased
 
+* Some attributes have been updated to align with
+  the latest version of Semantic Conventions:
+  - Rename `db.name` to `db.namespace`
+  - Rename `db.statement` to `db.query.text`
```

Comment:
I think that this part should be postponed according to https://github.com/open-telemetry/semantic-conventions/blob/v1.27.0/docs/database/database-spans.md Switch should be done as a batch.

Reply:
I'm fine with that. We can wait for the db stabilization.